### PR TITLE
MMT-2094 Delete Subscriptions

### PR DESCRIPTION
**Pull request by ryanmiller-1**

### CODE REVIEW

File: app/views/subscriptions/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<div class="row row-content">
+  <section>
+    <h2><%= @subscription.fetch('Name')%></h2>
+    <p>Collection Concept ID: <%= @subscription.fetch('CollectionConceptId') %></p>
+    <p>Query: <%= @subscription.fetch('Query') %></p>
+
+    <h3>Subscriber(s)</h3>
```
Comment (ryanmiller-1) [inline app/views/subscriptions/show.html.erb]: remove (s)

File: app/views/subscriptions/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          <td class="align-l"><%= @subscription['EmailAddress'] %></td>
+          <td class="align-l"><%= @subscription['SubscriberId'] %></td>
+        </tr>
+      </tbody>
+    </table>
+    <% if policy(:subscription).edit? %>
+      <%= link_to 'Edit', subscriptions_path, class: 'eui-btn' %>
+    <% end %>
+    <% if policy(:subscription).destroy? %>
+      <%= link_to 'Delete', '#delete-group-modal', class: 'display-modal eui-btn eui-btn--red' %>
```
Comment (ryanmiller-1) [inline app/views/subscriptions/show.html.erb]: subscription modal

File: spec/features/manage_cmr/subscriptions/show_subscription_spec.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
+describe 'Deleting Subscriptions' do
```
Comment (ryanmiller-1) [inline spec/features/manage_cmr/subscriptions/show_subscription_spec.rb]: show

File: app/controllers/subscriptions_controller.rb [hunk 4]
```diff
@@ -27,31 +27,64 @@ def new
...
-    subscription_response = cmr_client.ingest_subscription(reformat_subscription_params_for_submission.to_json, current_user.provider_id, "mmt_subscription_#{SecureRandom.uuid}", token)
+    subscription_response = cmr_client.ingest_subscription(reformated_subscription.to_json, current_user.provider_id, native_id, token)
     if subscription_response.success?
-      redirect_to subscriptions_path, flash: { success: 'Subscription was successfully created.'}
+      flash[:success] = I18n.t('controllers.subscriptions.create.flash.success')
+      redirect_to subscription_path(subscription_response.parsed_body['result']['concept_id'], subscription: reformated_subscription, native_id: native_id)
     else
       Rails.logger.error("Creating a subscription failed with CMR Response: #{subscription_response.clean_inspect}")
-      flash[:error] = subscription_response.error_message
+      flash[:error] = subscription_response.error_message(i18n: I18n.t('controllers.subscriptions.create.flash.success'))
```
Comment (jdf-eos) [inline app/controllers/subscriptions_controller.rb]: Should this be controllers.subscriptions.create.flash.error ?

Comment (ryanmiller-1) [reply app/controllers/subscriptions_controller.rb]: Yes

File: app/views/subscriptions/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,39 @@
+<div class="row row-content">
+  <section>
+    <h2><%= @subscription.fetch('Name')%></h2>
+    <p>Collection Concept ID: <%= @subscription.fetch('CollectionConceptId') %></p>
+    <p>Query: <%= @subscription.fetch('Query') %></p>
+
+    <h3>Subscriber</h3>
+    <table class="space-top space-bot" id="subscribers">
```
Comment (charleshuang80) [inline app/views/subscriptions/show.html.erb]: should we change the id to be singular?

File: app/controllers/subscriptions_controller.rb [hunk 6]
```diff
@@ -78,6 +109,10 @@ def get_subscriber_email(subscriber_id)
   end
 
   def reformat_subscription_params_for_submission
+    # CMR expects these parameters in 'ThisCase'.  In this iteration,
```
Comment (charleshuang80) [inline app/controllers/subscriptions_controller.rb]: we can edit this later if we want but it's 'PascalCase'

**Pull request merged**